internal/wait/daemonset.go

```diff
@@ -36,11 +36,12 @@ func (wt Waiter) ForDaemonSetReadyByKey(ctx context.Context, key ObjectKey) (*ap
 		}
 
 		if !AreDaemonSetPodsReady(&updatedDs.Status) {
-			klog.Warningf("daemonset %s desired %d scheduled %d ready %d",
+			klog.Warningf("daemonset %s desired %d scheduled %d ready %d up-to-date %d",
```

Comment:
this is an important fix and should probably be moved in its own PR. But let's keep it like this for now

Reply:
done: https://github.com/openshift-kni/numaresources-operator/pull/851